compiler/testData/diagnostics/tests/thisAndSuper/unqualifiedSuper/unqualifiedSuperWithAbstractMembers.fir.kt

```diff
@@ -25,7 +25,7 @@ class B : A(), I {
     override val y: Int = super.y
 
     override fun foo(): Int {
-        super.foo()
+        super.<!ABSTRACT_SUPER_CALL!>foo()<!>
```

Comment:
Please note that old front-end reports diagnostic on `foo` only, and here we have diagnostic on `foo()` with parentheses. It's worse, imagine some call like
```
super.foo(
    longArgument1,
    longArgument2,
    longArgument3,
    // ...
)
```

In IDEA you will have all these six lines underscored with red, it's too noisy. 

You may take `source` not from the whole call but from one of its children (I hope) to fix this.

Reply:
Got it. I'll fix my code